frontend/src/translations/result/en.json

```diff
@@ -420,6 +420,7 @@
   "welcome.how.workflow.main.title": "Workflow",
   "welcome.how.workflow.contrib.title": "For contributors",
   "payment.options.warning.message": "When you activate your bank account you will receive transfers directly. If the country that you have a bank account in is not on the list, it is possible to activate your account with PayPal.",
+  "user.confirm.password.error": "Passwords do not match",
```

Comment:
The result file is generate by our translation platform, Crowdin: 

So you should only add an entry here: https://github.com/worknenjoy/gitpay/blob/master/frontend/src/translations/en.json

And these files should not be changed